ibis/tests/sql/snapshots/test_compiler/test_column_expr_default_name/decompiled.py

```diff
@@ -0,0 +1,4 @@
+import ibis
+
+
+result = ibis.table(name="int_col_table", schema={"int_col": "int32"}).int_col + 4
```

Comment:
Should this be generating an intermediate table variable?

Reply:
Currently when an expression gets used once it doesn't generate an intermediate variable, though we can add exceptions based on the types.

Reply:
Ah, ok. I think this is fine as is.

Reply:
Updated.